common/src/main/java/com/wynntils/models/gear/GearModel.java

```diff
@@ -80,24 +85,49 @@ public boolean canBeGearBox(GearInfo gear) {
 
     @Override
     public void reloadData() {
-        gearInfoRegistry.reloadData();
+        gearInfoRegistry.loadData();
     }
 
+    // For "real" gear items eg. from the inventory
     public GearInstance parseInstance(GearInfo gearInfo, ItemStack itemStack) {
         WynnItemParseResult result = WynnItemParser.parseItemStack(itemStack, gearInfo.getVariableStatsMap());
         if (result.tier() != gearInfo.tier()) {
             WynntilsMod.warn("Tier for " + gearInfo.name() + " is reported as " + result.tier());
         }
 
+        // probably migrate this to GearInstance?
```

Comment:
Let's migrate this to the call just above this one :) `WynnItemParser#parseItemStack` already does the requirements parsing, adding a check if just a matter of adding/checking a group in the regex. You can return the value in `WynnItemParseResult`.